Neuro/Pathfinding/DataStructures/Node.cs

```diff
@@ -1,40 +1,19 @@
 ﻿using UnityEngine;
 
-namespace Neuro.Utils;
+namespace Neuro.Pathfinding.DataStructures;
 
 public class Node : IHeapItem<Node>
```

Comment:
since this class doesn't have behavior, it should be a C# record type.

Reply:
I do not wish to modify `Node.cs` as part of this pull request.